storage-proofs/src/vde.rs

```diff
@@ -126,3 +127,28 @@ pub fn create_key<H: Hasher>(
     let hash = hasher.finalize();
     Ok(bytes_into_fr_repr_safe(hash.as_ref()).into())
 }
+
+// FIXME: CHECK THIS.
```

Comment:
Seems okay to me, what exactly do you want to check?

Reply:
The existence of the function itself mainly, I introduced it to avoid accesing slices directly and wanted to raise attention to it to validate if this was in alignment with the current API we want to build here.

I'll remove the note then.